docs/integrations/open_telemetry.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: OpenTelemetry Integration Guide
```

Comment:
This is great. I think we should move it to another section in docs, maybe under `setup`. Keeping it in `integrations` may confuse the users with the actual data sources mindsdb works with.

Reply:
Agreed!